topic5/movies/src/index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import registerServiceWorker from './registerServiceWorker';
+
+ReactDOM.render(<App />, document.getElementById('root'));
+registerServiceWorker();
```

Comment:
Do you really need this service worker? You could put it inside App component. 
Where? under the componentDidMount() function. 

Once App is rendered, you call this registerServiceWorker() function